### DISCUSSION TITLE
wio wm 1110 lorawan network server

### DISCUSSION THREAD

**Discussion by jishnump** [Seeed-Solution/SenseCAP_Indicator_ESP32#39] [General]
Does this wio wm 1110 dev kit sample code for other  lorawanwan network server (eg : semtech lorawan network server)?
the current sample code is for TTN as per the doc

**Comment by Love4yzp** [comment]
I could not understantd what you want, could you put on more detail, what is your question, what you need?

**Comment by jishnump** [comment]
Hi,

Currently I am using wio wm 1110 dev kit and following official doc of that
board. It guide to setup the device to connect with TTN lorawan network
server. So I need to know with using the same sample code can we connect
this device to other lorawan network server (semtech lorawan network server)

Regards,
Jishnu M P

**Reply by Love4yzp** [reply]
The gateway will forward the LoRaWAN package to your LNS, you’ll need to register the credentials on your LNS.
So the code is ok, as the LNS has the credentials of your device.
